L/LibSSH2/build_tarballs.jl

```diff
@@ -35,6 +38,11 @@ mkdir build && cd build
 cmake .. "${BUILD_FLAGS[@]}"
 make -j${nproc}
 make install
+
+# copy import lib `libssh2.dll.a` => `liblibssh2.dll.a` to maintain backwards compatibility
+if [[ ${target} == *-mingw* ]]; then
+    install -Dvm 0755 "src/libssh2.dll.a" "${prefix}/lib/liblibssh2.dll.a"
+fi
```

Comment:
I'm not really sure we care about keeping compatibility, also because import libraries are relevant only at build time, not runtime.  @staticfloat opinions?

Reply:
Yeah, I don’t think this will break anything. Especially since libssh2 moves in lock step with Julia, since it’s an stdlib.

Reply:
Done.